app/models/user.rb

```diff
@@ -5,6 +5,8 @@ class User < ApplicationRecord
   # :confirmable, :trackable, :omniauthable and :rememberable
   devise :database_authenticatable, :recoverable, :validatable, :lockable, :timeoutable
 
+  enum role: { user: 0, admin: 1 }, _suffix: :role
+
```

Comment:
wo genau wird definiert, dass der admin_user alles lesen und schreiben darf ?

Reply:
In den neuen scopes der einzelnen Resourcen